features/202-session-management/config.md

```diff
@@ -0,0 +1,51 @@
+# Session Configuration
+
+## Sample configuration
+
+In user configuration:
+```yaml
+clients:
+    - name: Web App
+      disabled: false
+      api_key: XXX
+      session_transport: cookie
+    - name: iOS App
+      disabled: false
+      api_key: YYY
+      session_transport: header
+    - name: Android App
+      disabled: false
+      api_key: ZZZ
+      session_transport: header
+session:
+    access_token_lifetime: 1800
+    refresh_token_disabled: false
+    refresh_token_lifetime: 86400
+    session_idle_timeout_enabled: true
+    session_idle_timeout: 300
```

Comment:
A more configurable design is to inline the session configuration into client configuration. Each client configuration can have its own session configuration. Notably when `session_transport: cookie`, `refresh_token_*` are errors.

Reply:
Make sense.

But I prefer to have each API key able to do both cookies and header based session.